trade-finance-manager-api/src/v1/services/deal-cancellation/deal-cancellation.service.ts

```diff
@@ -71,6 +71,7 @@ export class DealCancellationService {
       throw new Error(`Some UKEF facility ids were invalid when submitting deal ${dealId} for cancellation. No email has been sent`);
     }
 
-    await this.sendDealCancellationEmail(cancelledDealUkefId.toString(), cancellation, riskExpiredFacilityUkefIds);
+    // TODO: fix this once getUkefDealId is merged in
```

Comment:
TODO - must wait til this is in

Reply:
I'm not understanding what this means - do we have a ticket number?

Reply:
Ah this is a note to myself! I've merded in the PR with `getUkefDealId` now so I'll update this & push the changes